src/dotnet/commands/dotnet-build/README.md

```diff
@@ -1,38 +1,87 @@
-dotnet-build
-===========
+% DOTNET-BUILD(1)
+% Zlatko Knezevic zlakne@microsoft.com
```

Comment:
Change the author to "Microsoft Corporation" and use a feedback DL for the email address?